requirements.txt

```diff
@@ -12,5 +12,5 @@ configargparse>=1.2.3
 py-multihash>=0.2.3
 multiaddr @ git+https://github.com/multiformats/py-multiaddr.git@e01dbd38f2c0464c0f78b556691d655265018cce
 cryptography>=3.4.6
-pydantic>=1.8.1,<2.0
-packaging>=20.9
+pydantic>=2.0.0
+packaging>=20.9
```

Comment:
Looks like the file is missing a line break at the end, could you please add it back?

Reply:
Done.